src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1135,14 +1139,17 @@ private function registerDebugConfiguration(array $config, ContainerBuilder $con
     {
         $loader->load('debug_prod.php');
 
+        $debug = $container->getParameter('kernel.debug');
```

Comment:
not needed

Reply:
I've just moved it, it is already used many times below.